ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python
+
+# Copyright 2016 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Created by Vikram Chandrashekhar.
+# Edited by Greg Kiar, Eric Bridgeford and Chuankai Luo.
+# Email: Greg Kiar @ gkiar@jhu.edu   cluo16@jhu.edu
+
+import warnings
+
+warnings.simplefilter("ignore")
+import os
+import re
+import sys
+import numpy as np
+import nibabel as nb
+# import ndmg.utils as mgu
+from argparse import ArgumentParser
+from scipy import ndimage
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib as mpl
+
+mpl.use("Agg")  # very important above pyplot import
+from nilearn.plotting.edge_detect import _edge_map as edge_map
+import matplotlib.pyplot as plt
+
+
+
+
+def gen_overlay_pngs(
+    brain, origional, outdir, loc=0, mean=False, minthr=2, maxthr=95, edge=False):
+    """
+    outdir: directory where output png file is saved
+    fname: name of output file WITHOUT FULL PATH. Path provided in outdir.
+    """
+    try:
+        origional_name = get_filename(origional)
+    except:
+        origional_name = 'compare_origional_data_and_brain'
+    brain_data = nb.load(brain).get_data()
+    if brain_data.ndim == 4:  # 4d data, so we need to reduce a dimension
+        if mean:
+            brain_data2 = brain_data.mean(axis=3)
+        else:
+            brain_data2 = brain_data[:, :, :, loc]
+    else:  # dim=3
+        brain_data2 = brain_data
+
+    cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["white", "magenta"])
+    cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["white", "green"])
+
+
+    fig = plot_overlays(brain_data2, origional, [cmap1, cmap2], minthr, maxthr, edge)
+
+    # name and save the file
+    fname = "qa_skullstrip__" + origional_name + ".png"
+    fig.savefig(outdir + "/" + fname, format="png")
+    # plt.close()
+
+
+def opaque_colorscale(basemap, reference, vmin=None, vmax=None, alpha=1):
+    """
```

Comment:
function already exists in ndmg.stats.qa_reg

Reply:
fixed it in the latest version